### PR TITLE
docs(examples): setup get started with examples guide

### PR DESCRIPTION
**Pull request by arjunattam**
Follow up PRs to add more examples, add a section explaining core concepts (like browser contexts), and maybe elaborate on the dev environment.

Would love to get your feedback.

Related to #660

### CODE REVIEW

**Review by yury-s — APPROVED** [review]
Awesome!

File: docs/examples/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Playwright is a Node.js library and can be acquired through the npm registry. Use npm or yarn to install Playwright in your Node.js project.
+
+```
+npm i playwright
+```
+
+Once installed, you can `require` Playwright in your Node.js scripts. 
+
+```js
+const playwright = require('playwright');
```
Comment (dgozman) [inline docs/examples/README.md]: We recommend destructuring everywhere: `const { chromium } = require('playwright');`.

File: docs/examples/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+(async () => {
+  const browser = await playwright.chromium.launch();
+  await browser.close();
+})();
+```
+
+## Setup dev environment
+
+Playwright scripts can be developed just like any other Node.js script. You can use the [Node.js debugger](https://nodejs.org/api/debugger.html) or [VS Code debugging](https://code.visualstudio.com/docs/nodejs/nodejs-debugging) to set breakpoints and get fine grained control over execution.
```
Comment (dgozman) [inline docs/examples/README.md]: Let's say 'for example' to not imply we only work with these two debuggers.

File: docs/examples/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+### Running browsers for debugging
+
+By default, Playwright runs the browsers in headless mode. To see the browser UI, pass the `headless: false` flag while launching the browser. You can also use `slowMo` to slow down execution.
+
+```js
+  playwright.firefox.launch({ headless: false, slowMo: 50 });
```
Comment (dgozman) [inline docs/examples/README.md]: - `chromium.launch(...)`
- Let's use chromium everywhere for consistency, and mention at the top that all examples use Chromium, but can be similarly run with Firefox/WebKit?

File: docs/examples/README.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+* Browser
+* Browser contexts
+* Pages and frames
+* Evaluate in page context
+-->
+
+<!--
+Other examples
+* Page navigation and wait for load
```
Comment (dgozman) [inline docs/examples/README.md]: * Filling a form
* Uploading a file
* Handling a popup, e.g. clicking "I accept" to close it

File: docs/examples/authentication.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  // Navigate to GitHub.com and verify that we are logged in
+  const wkPage = await wkContext.newPage();
+  await wkPage.goto('http://github.com');
+  await wkPage.screenshot({ path: 'webkit.png' });
+  await verifyIsLoggedIn(wkPage);
+  await wkBrowser.close();
+})();
+
+const verifyIsLoggedIn = async (page) => {
+  await page.waitForSelector('"Create repository"');
```
[the remark this answers is not in the document]
Comment (arjun27) [reply docs/examples/authentication.js]: Thanks! I moved to use `page.$` but perhaps there is a better solution.

```js
const verifyIsLoggedIn = async (page) => {
  // Find element through text value and assert it exists.
  assert(await page.$('text="Create repository"'));
}
```

**Pull request merged**